spec/factories/users.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  factory :user do
+    name {'taro'}
+    email {'abc@gmail.com'}
```

Comment:
存在しないことが保証されているドメインを使う様にしてください。
メール送信のテストなどで、実際に存在する相手にメールが送られてしまう、などのリスクがあります。
```suggestion
    email {'abc@example.com'}
```